avaframe/com1DFAPy/com1DFA.py

```diff
@@ -111,7 +111,7 @@ def com1DFAMain(cfg, avaDir, relTh):
             relFiles = releaseLine['file']
             # ------------------------
             #  Start time step computation
-            Tsave, T, U, Z, S, Particles, Fields, Tcpu = DFAIterate(cfgGen, particles, fields, dem)
+            Tsave, T, U, Z, S, Particles, Fields, Tcpu = DFAIterate(cfgGen, particles, fields, dem, avalancheDir, logName)
```

Comment:
Maybe we should use a general `info` dict that is being returned, with eg. the Tcpu, the additional mass info + any other (potential)info in it. -> Then we could drop the avalanchDir and logName input...

Reply:
Particles, the list of particles dictionaries contains all the mass info, however only for the saving time steps, however aimec requires the mass balance info to be listed for every computation time step. For this reason, it is written to file within the function DFAiterate

Reply:
changed